README.md

```diff
@@ -74,6 +74,59 @@
 | 7 | u8 | dst_device_id | The device ID of the intended recipient of the message. |
 | 8-n | u8[] | payload | The message payload. |
 | (n+1)-(n+2) | u16 | checksum | The message checksum. The checksum is calculated as the sum of all the non-checksum bytes in the message. |
+
+## Pack and unpack example
+
+To request a message from the sensor, it's necessary to use [`general_request` message](#6-general_request) to request a specific message ID, and to build the message, the only thing necessary is to follow the [format description](https://github.com/bluerobotics/ping-protocol#format).
+In this example, will be demonstrated how to request a message, pack the bytes, unpack and extract the necessary information.
+
+##### Pack
+
+1. First it's necessary to defined the message that we want, in this case `distance_simple`.
+2. The message ID of `general_request` is 6, so this value should be used in `message_id` field.
+3. Usually the master [parent],  will use the ID 0 and the slave [worker], in that case Ping, will boot with ID 1. This values should be used in `src_device_id` and `dst_device_id` respectively. It's also possible to use `dst_device_id` as 0 to do a broadcast.
```

Comment:
src/dst id is not implemented in firmware yet, correct?

Reply:
Nops.

Reply:
```
m_head.src_device = 0;
m_head.dst_device = 0;
```